src/main/java/uk/gov/hmcts/darts/transcriptions/service/impl/TranscriptionServiceImpl.java

```diff
@@ -136,6 +143,8 @@ public class TranscriptionServiceImpl implements TranscriptionService {
     private final YourTranscriptsQuery yourTranscriptsQuery;
     private final DuplicateRequestDetector duplicateRequestDetector;
     private final TranscriberTranscriptsQuery transcriberTranscriptsQuery;
+    private final EntityManager em;
```

Comment:
Is this variable used anywhere in this class?

Reply:
have now removed.